dts/arc/arc_axs.dtsi

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2019, Synopsys, Inc. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "skeleton.dtsi"
+
+#include <dt-bindings/i2c/i2c.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "snps,archs48";
```

Comment:
AXS103 has HS3x not HS4x

Reply:
AXS103 1.3 version support HS4x.

Reply:
Well in fact we have 2 flavors of AXS103:
1. FPGA firmware v1.2: it has ARC HS38x2
2. FPGA firmware v1.3: it has ARC HS48x2 or ARC HS47D (depending on used .bit-file)

But if we don't need DSP extensions we may treat both v1.2 with HS38x2 & v1.3 with HS48x2 as the same core because from software perspective they are very similar. And if we need to do something specific per core flavor we may do so in run-time based on read IDENTITY register.

Though given we're discussing very optimized RTOS we may still prefer to have 2 a bit separate boards like "axs103_hs38x2" & "axs103_hs48x2" because actually AXS103 v1.2 has also ARC HS36 in FPGA configuration (and is selectable via dip-switch on the board) so there might be 3rd "Zephyr board" like "axs103_hs36" (note it's a single-core configuration).